client/src/app/videos/+video-edit/shared/video-edit.component.ts

```diff
@@ -98,7 +98,8 @@ export class VideoEditComponent implements OnInit, OnDestroy {
       thumbnailfile: null,
       previewfile: null,
       support: this.videoValidatorsService.VIDEO_SUPPORT,
-      schedulePublicationAt: this.videoValidatorsService.VIDEO_SCHEDULE_PUBLICATION_AT
+      schedulePublicationAt: this.videoValidatorsService.VIDEO_SCHEDULE_PUBLICATION_AT,
+      producedAt: this.videoValidatorsService.VIDEO_PRODUCED_AT
```

Comment:
We may add in this file an `onProducedAtAdded()` function to check if the given date is prior to the creation date.

Reply:
no doubt it will always be produced before published, unless there was an error on the part of the uploader or some kind of bug, so lets not make it fail in that case, maybe throw up a warning, to notify about it but to stop it.

Reply:
We were indeed thinking about some error the user could make when setting this field. OK for displaying a warning.